README.rst

```diff
@@ -1002,6 +1002,10 @@ Choose from the list of available rules:
 
   *Risky rule: modifies the signature of functions; therefore risky when using systems (such as some Symfony components) that rely on those (for example through reflection).*
 
+* **no_unset_on_property**
```

Comment:
please add this rule to local `.php_cs.dist` file

Reply:
Done